metricflow/sql/render/databricks.py

```diff
@@ -0,0 +1,49 @@
+from metricflow.object_utils import assert_values_exhausted
+from metricflow.sql.render.expr_renderer import (
+    DefaultSqlExpressionRenderer,
+    SqlExpressionRenderer,
+    SqlExpressionRenderResult,
+)
+from metricflow.sql.render.sql_plan_renderer import DefaultSqlQueryPlanRenderer
+from metricflow.sql.sql_exprs import SqlPercentileExpression, SqlPercentileFunctionType
+
+
+class DatabricksSqlExpressionRenderer(DefaultSqlExpressionRenderer):
+    """Expression renderer for the Databricks engine."""
+
+    def visit_percentile_expr(self, node: SqlPercentileExpression) -> SqlExpressionRenderResult:
+        """Render a percentile expression for Databricks."""
+        arg_rendered = self.render_sql_expr(node.order_by_arg)
+        params = arg_rendered.execution_parameters
+        percentile = node.percentile_args.percentile
+
+        if node.percentile_args.function_type is SqlPercentileFunctionType.CONTINUOUS:
+            function_str = "PERCENTILE_CONT"
+        elif node.percentile_args.function_type is SqlPercentileFunctionType.DISCRETE:
+            function_str = "PERCENTILE_DISC"
+        elif node.percentile_args.function_type is SqlPercentileFunctionType.APPROXIMATE_CONTINUOUS:
+            raise RuntimeError(
+                "Approximate continuous percentile aggregate not supported for Snowflake. Set use_discrete_percentile to true and/or use_approximate_percentile to false in all measures."
```

Comment:
nit: copy/paste error Snowflake -> Databricks. Also same thing as last one.
```suggestion
                "Approximate continuous percentile aggregate not supported for Databricks. Set "
                "use_discrete_percentile to true and/or use_approximate_percentile to false in all percentile measures."
```